README.md

```diff
@@ -1,65 +1,58 @@
 # GitHub Training Kit
+### Home of GitHub's On-Demand training
 
-This is the official courseware for the [GitHub Training Team](http://training.github.com). This repository provides open source materials and slides for teaching GitHub Classes under the [_CC BY 4.0_ license](http://creativecommons.org/licenses/by/4.0/).
+This repository contains the completely open source on-demand training hosted at https://services.github.com/on-demand/. These materials are provided under a [_Creative Commons License_ license](https://github.com/github/training-kit/blob/master/LICENSE).
```

Comment:
🤔 This is probably my naïveté on the subject, but do we need to include the word license twice?